### PR TITLE
O3-3791: Dispensing: display instructions from DrugOrder.instructions…

### PR DESCRIPTION
**Pull request by mogoodrich**
… when dispensing drugs

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

File: src/medication-dispense/medication-dispense.resource.tsx [hunk 1]
```diff
@@ -135,7 +135,16 @@ export function initiateMedicationDispenseBody(
       whenHandedOver: null,
       dosageInstruction: [
         {
-          text: medicationRequest.dosageInstruction[0].text,
```
Comment (mseaton) [inline src/medication-dispense/medication-dispense.resource.tsx:138]: This is sort of ugly, I'd imagine there might be a better way to do this, but if it works, that's fine I guess.

**Review by mseaton — APPROVED** [review]
Looks fine to me, though one comment

File: src/medication-dispense/medication-dispense.resource.tsx (hunk 1 above)
Comment (mogoodrich) [reply src/medication-dispense/medication-dispense.resource.tsx:138]: It's definitely more than sort of ugly, agreed.... I was hoping someone might have a "duh" better way to do it, but I can also take another stab before merging.

**Review by PiusKariuki — APPROVED** [review]

File: src/medication-dispense/medication-dispense.resource.tsx (hunk 1 above)
Comment (mogoodrich) [reply src/medication-dispense/medication-dispense.resource.tsx:138]: Made this simpler.

**Pull request merged**